dipy/nn/tests/test_finta_model.py

```diff
@@ -0,0 +1,173 @@
+import os
+import pytest
+import tempfile
+
+import numpy as np
+import numpy.testing as npt
+
+import dipy.nn.utils as utils
+from dipy.utils.optpkg import optional_package
+
+tf, have_tf, _ = optional_package("tensorflow", min_version="2.0.0")
+
+if have_tf:
+    from dipy.nn.finta_model import Encoder, Decoder
+    from dipy.nn.finta_model import (
+        IncrFeatStridedConvFCUpsampReflectPadAE as AE)
+
+
+class DummyData():
```

Comment:
Is there a way to use one of the existing datasets; the smallest one? Or one of the DiSCo dataset bundles?

Reply:
Resolved as commented in the weekly meeting.
Conclusion -> We stick to the DummyData since it is deterministic and fast to generate.